tests/test_topas_converter.py

```diff
@@ -71,12 +73,17 @@ def parser() -> Parser:
     """Just a parser fixture."""
     return get_parser_from_str('topas')
 
+@pytest.fixture
+def default_json() -> dict:
+    """Creates default json."""
+    with open(path.join(path.abspath("./input_examples"), 'topas_parser_test.json'), 'r') as json_f:
```

Comment:
use python pathlib instead of path module.

Reply:
ok